web/src/pages/licenses/[licenseUrlSlug].tsx

```diff
@@ -71,27 +69,29 @@ export const LicenseElement = (props: LicenseElementProps): ReactElement => {
 };
 
 interface Props {
-  license: LicenseEvent;
+  license: LicenseEventType;
   licenseEventType: LicenseEventSubtype;
 }
 
 const LicensePage = (props: Props): ReactElement => {
   const { business } = useUserData();
-  let date = parseDate(business?.licenseData?.expirationISO);
+  const licenseName = props.license.licenseName;
+
+  const expirationDateISO = business?.licenseData?.licenses?.[licenseName]?.expirationDateISO;
```

Comment:
dust - 
![image](https://github.com/user-attachments/assets/b0e96e5c-d38c-4a00-a888-911c39e69540)
I was getting an type error for this in my ide, is it going insane or is there some type fenangling we coudl or should do